book/src/external-snapshot-metadata.md

```diff
@@ -0,0 +1,118 @@
+# CSI external-snapshot-metadata
+
+## Status and Releases
+
+**Git Repository:** [https://github.com/kubernetes-csi/external-snapshot-metadata](https://github.com/kubernetes-csi/external-snapshot-metadata)
+
+### Supported Versions
+
+Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](project-policies.md#minimum-version) | [Max K8s Version](project-policies.md#maximum-version) | [Recommended K8s Version](project-policies.md#recommended-version) |
+--|--|--|--|--|--|--|--
+v0.1.0 | [v0.1.0](https://github.com/kubernetes-csi/external-snapshot-metadata/releases/tag/v0.1.0) | [v1.10.0](https://github.com/container-storage-interface/spec/releases/tag/v1.10.0) | - | registry.k8s.io/sig-storage/csi-snapshot-metadata:v0.1.0 | v1.32 | - | v1.32
```

Comment:
https://github.com/kubernetes-csi/external-snapshot-metadata/releases/tag/v0.1.0 is not available yet.  We need to have the release out first before merging the doc.